wxHexEditor.appdata.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Submit this upstream.

Reply:
For the v0.24 release this will have to stay here.
When updating this to the next version this will be thrown out for sure.
I will create PR for that together with a few more things to get this in line with standards when activity upstream resumes.